vcbuild.bat

```diff
@@ -324,7 +324,10 @@ if errorlevel 1 (
 if "%target%" == "Clean" goto exit
 
 :after-build
+rd out\%config%
```

Comment:
If `out\%config%` exist but is just a symlink, `rd` will succeed.

Reply:
uh isn't `%config%` the symbolic link and `out\%config%` the build artifacts?

Reply:
🤦‍♂️

Reply:
I just took the second argument from the `mklink` line.
So facepalm on MS for switching the order of args.